corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -123,17 +123,11 @@ function LocationTypeModel(data, root) {
 
     this.validate = function() {
         this.name_error(false);
-
-        valid = true;
-
         if (!this.name()) {
-            this.name_error(false);
-            valid = false;
-        }
-            valid = false;
+            this.name_error(true);
```

Comment:
the bad merge was the duplicated
```javascript
    valid = false;
}
```
Which caused a syntax error.  The `false` to `true` was a code error I noticed while cleaning this up (it wouldn't actually display the error message)